Sources/joyfill-components-swift/Components/Chart/ChartView.swift

```diff
@@ -332,6 +343,16 @@ public class ChartView: UIViewController, UITextFieldDelegate, ChartViewTextFiel
         let tap = UITapGestureRecognizer(target: self, action: #selector(showHideButtonTapped))
         showHideButtonView.addGestureRecognizer(tap)
         
+        if #available(iOS 13.0, *) {
```

Comment:
Can you add a comment to what exactly this if statement is solving for

Reply:
Done.